core/src/types/key/secp256k1.rs

```diff
@@ -499,6 +551,10 @@ impl TryFrom<&[u8; 65]> for Signature {
 pub struct SigScheme;
 
 impl super::SigScheme for SigScheme {
+    #[cfg(any(test, feature = "secp256k1-sign-verify"))]
+    type Hasher = crate::ledger::storage::KeccakHasher;
+    #[cfg(not(any(test, feature = "secp256k1-sign-verify")))]
+    type Hasher = crate::ledger::storage::DummyHasher;
```

Comment:
should this impl block exist at all if the sign-verify flag is disabled?

Reply:
it exists, if you notice the code in `sign()`, we panic there

Reply:
But *should* it? Instead of runtime panics, we could have compile time checks. Is this trait bound required elsewhere in note feature gated code?